libs/wingsdk/src/core/app.ts

```diff
@@ -188,6 +188,11 @@ export abstract class CdktfApp extends App {
 
     super(cdktfStack, "Default");
 
+    // TODO: allow the user to specify custom backends
+    new cdktf.LocalBackend(cdktfStack, {
+      path: "./terraform.tfstate",
```

Comment:
Crossing my fingers this works on Windows? 🤞 Terraform CLI is built with Go so it probably should...

I wanted to use `path.join(outdir, "terraform.tfstate")` here, but that resulted in a path like `/path/to/project-dir/target/test/main.tfaws.036281.tmp/terraform.tfstate` getting generated 😕

Reply:
(To test this in CI we would actually have to run `terraform apply` in our tests, and we don't have the tooling set up for that yet...)

Reply:
Might be interesting to add https://developer.hashicorp.com/terraform/cli/commands/validate to hangar (and/or some part of wing's toolchain)

Reply:
I think at some point we ran validate as part of the SDK tests, the problem was just that it's very slow (not sure what they're doing under the hood)